### PR TITLE
fix: use consistent monospace fonts HELP-52113

### PR DESCRIPTION
**Pull request by lerouxb**

### CODE REVIEW

File: packages/compass-crud/src/components/table-view/document-table-view.less [hunk 1]
```diff
@@ -61,8 +61,8 @@
     }
 
     &-parent {
-      font-family: Menlo, Monaco, Consolas, 'Courier New', monospace;
-      font-size: 11px;
+      font-family: 'Source Code Pro', Menlo, monospace;
```
Comment (Anemy) [inline packages/compass-crud/src/components/table-view/document-table-view.less:64]: Is `Source Code Pro` a font we load somewhere?

**Review by Anemy — APPROVED** [review]
lgtm, question on where the font is loaded (or is it one that a lot of operating systems have?)

File: packages/compass-crud/src/components/table-view/document-table-view.less (hunk 1 above)
Comment (gribnoysup) [reply packages/compass-crud/src/components/table-view/document-table-view.less:64]: Loaded in renderer entrypoint: https://github.com/mongodb-js/compass/blob/53ac5525a645befb1f311193d81f6dd6d17510e2/packages/compass/src/app/index.jsx#L34

Comment (Anemy) [reply packages/compass-crud/src/components/table-view/document-table-view.less:64]: ty!

**Pull request merged**